src/lib/ikea.ts

```diff
@@ -825,3 +838,87 @@ export function addCustomClusterManuSpecificIkeaUnknown(): ModernExtend {
         commandsResponse: {},
     });
 }
+
+const unfreezeMechanisms: {
+    [key: string]: (entity: Zh.Endpoint | Zh.Group) => Promise<void>;
+} = {
+    // WS lights:
+    //   Aborts the color transition midway: light will stay at the intermediary
+    //   state it was when it received the command.
+    // Color lights:
+    //   Do not support this command.
+    moveColorTemp: async (entity) => {
+        await entity.command("lightingColorCtrl", "moveColorTemp", {rate: 1, movemode: 0, minimum: 0, maximum: 600}, {});
+    },
+
+    // WS lights:
+    //   Same as "moveColorTemp".
+    // Color lights:
+    //   Finishes the color transition instantly: light will instantly
+    //   "fast forward" to the final state, post-transition.
+    genLevelCtrl: async (entity) => {
+        await entity.command("genLevelCtrl", "stop", {}, {});
+    },
+};
+
+// zigbee commands which will freeze an IKEA light
+const willFreeze = (clusterKey: string | number, payload: KeyValue): payload is KeyValue & {transtime: number} =>
+    payload &&
+    // any color command with a transition will freeze the light...
+    (payload.transtime as number) > 0 &&
+    clusterKey === "lightingColorCtrl" &&
+    // ...except for 'stop' commands:
+    payload.rate !== 1 &&
+    payload.movemode !== 0;
+
+// keys for whom we need to unfreeze the light before issuing a corresponding command to said light
+const keysNeedingUnfreeze = new Set(["state", "brightness", "brightness_percent", "color_temp", "color_temp_percent"]);
+
+const ikea_bulb_unfreeze = (next: Tz.Converter["convertSet"]) => {
```

Comment:
Could you explain (very short) what the problem is and why this solution fixes it, also add a link to the issue (for future reference)

Reply:
Sure - done